### PR TITLE
Simplify quickstart by removing unneeded step

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

Reduce the number of steps in quickstart to be

- install control plane
- install a piped agent
- register an application
- deploy application
- clean up

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 1]
```diff
@@ -19,28 +19,26 @@ This guides you to install PipeCD in your kubernetes and deploy a `helloworld` a
...
-PipeCD comes with an embedded web-based UI.
-First up, using kubectl port-forward to expose the installed control-plane on your localhost:
+Once installed, use `kubectl port-forward` to expose the web console on your localhost:
 
 ``` console
 kubectl -n pipecd port-forward svc/pipecd 8080
 ```
 
-Point your web browser to [http://localhost:8080](http://localhost:8080) to login with the configured static admin account.
+Then point your web browser to [http://localhost:8080](http://localhost:8080) to login with the configured static admin account as below:
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/quickstart/_index.md:34]:
```suggestion
The PipeCD web console will be available at [http://localhost:8080](http://localhost:8080). To login, you can use the configured static admin account as below:
```
nits

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: >
   This page describes how to quickly get started with PipeCD on Kubernetes.
 ---
 
-This guides you to install PipeCD in your kubernetes and deploy a `helloworld` application to that Kubernetes cluster. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
+This guides you to install PipeCD in your kubernetes and deploy a "hello world" application to that Kubernetes cluster. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/quickstart/_index.md:9]:
```suggestion
This page is a guideline for installing PipeCD in your Kubernetes cluster and deploying a "hello world" application to that same Kubernetes cluster.

Note that it __does not require installing PipeCD to your all clusters to enable using PipeCD to deploy your applications__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```
nits

**Comment by khanhtc1202** [comment]
Nice
/lgtm

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ description: >
   This page describes how to quickly get started with PipeCD on Kubernetes.
 ---
 
-This guides you to install PipeCD in your kubernetes and deploy a `helloworld` application to that Kubernetes cluster. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
+This page is a guideline for installing PipeCD in your Kubernetes cluster and deploying a "hello world" application to that same Kubernetes cluster.
+
+Note that it __does not require installing PipeCD to your all clusters to enable using PipeCD to deploy your applications__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```
Comment (nakabonne) [inline docs/content/en/docs-dev/quickstart/_index.md:11]: nit: It took me a little time to get it.
```suggestion
Note that it __does not require installing PipeCD to all clusters where you deploy your applications typically__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```

Comment (khanhtc1202) [reply docs/content/en/docs-dev/quickstart/_index.md:11]: That's better 👍

**Comment by nakabonne** [comment]
Nice!
/approve

**Pull request merged**